micrometer-observation/build.gradle

```diff
@@ -39,19 +39,15 @@ jar {
 dependencies {
     api project(":micrometer-commons")
 
-    optionalApi 'io.micrometer:context-propagation'
+    optionalApi libs.contextPropagation
 
     // HttpServlet KeyValueProvider
-    optionalApi 'javax.servlet:javax.servlet-api'
+    optionalApi libs.javax.servletApi
 
     // Aspects
     optionalApi libs.aspectjrt
 
-    // log monitoring
-    testImplementation 'ch.qos.logback:logback-classic'
-    testImplementation 'org.apache.logging.log4j:log4j-core'
-
-    testImplementation project(':micrometer-observation-test')
```

Comment:
I'm surprised the build didn't fail with this. It's a circular dependency because micrometer-observation-test depends on this project (micrometer-observation).